Add GreenEarthOS Project Card and Page

grhhhh opened the pull request:
Fixes #2064

  - Adds green-earth-os.md to the _projects folder using information provided by the GreenEarthOS team [here](https://github.com/hackforla/GreenEarthOS/issues/3#issuecomment-893120885)
  - Adds [green-earth-os-hero.png and green-earth-os.png](https://github.com/hackforla/GreenEarthOS/issues/3#issuecomment-884618633) to the assets>images>projects folder
  
Adding the GreenEarthOS project markdown file as well as the GreenEarthOS image files creates a new project card and new project site for the GreenEarthOS project on the website.

<img width="50%" alt="Screen Shot 2021-08-14 at 4 09 01 PM" src="https://user-images.githubusercontent.com/79604590/129462958-869e7391-1690-4de8-a646-582e528dda92.png">
<img width="75%" alt="Screen Shot 2021-08-14 at 4 10 31 PM" src="https://user-images.githubusercontent.com/79604590/129462960-1e698b6b-5533-44f0-af41-980e2324ed20.png">

grhhhh committed:
Merge pull request #1 from hackforla/gh-pages

syncing my fork to original repository

grhhhh committed:
Merge pull request #2 from hackforla/gh-pages

Syncing my fork to the original repository

grhhhh committed:
adds green-earth-os.md file and GreenEarthOS image files

blakes24 commented on _projects/green-earth-os.md:
```diff
@@ -0,0 +1,55 @@
+description: The GreenEarthOS project seeks to create and publish a map of the current climate project support landscape including gaps. Once the landscape has been created the project will work to create visible guidance for open source projects and project ideators on how to navigate the landscape and connect with players at every appropriate stage (recruiting, mentoring, partnerships, packagin ...
+image: /assets/images/projects/green-earth-os.png
+alt: 'GreenEarthOS logo with text in green and a globe as a "O"'
+image-hero: /assets/images/projects/green-earth-os.png
```
Should image-hero be `green-earth-os-hero.png`?  Looks like you referenced `green-earth-os.png` twice, but never used `green-earth-os-hero.png`

grhhhh replied:
@blakes24 Thanks for catching this! My most recent commit fixes this issue.

blakes24 commented on _projects/green-earth-os.md:
```diff
@@ -0,0 +1,55 @@
+    links:
+      slack: 'https://hackforla.slack.com/team/UE1UG1YFP'
+      github: 'https://github.com/experiementsinhonesty'
+    picture: 
```
To get a user's picture you just need  to add `https://avatars.githubusercontent.com/<insert github name here>`

grhhhh replied:
@blakes24 Thanks for your help with this! My most recent commit addresses this.

blakes24 reviewed (changes requested):
Looks good, there are just a few issues I noticed.
- The text on the card doesn't line up with the other cards. It looks like all the other project images have a 3:2 ratio while the green earth image is 4:3, if you crop out some of the white space it should line up with the rest of the cards
- The links for the readme and project overview are missing
- The pictures for the project leaders are missing

Aveline-art commented:
> 
> 
> Looks good, there are just a few issues I noticed.
> 
>     * The text on the card doesn't line up with the other cards. It looks like all the other project images have a 3:2 ratio while the green earth image is 4:3, if you crop out some of the white space it should line up with the rest of the cards
> 
>     * The links for the readme and project overview are missing
> 
>     * The pictures for the project leaders are missing

@grhhhh Just to let you know, the images comes with a specific ratio (or, more accurately, a specific size) that must be met to appear properly aligned, as per @blakes24 . You need to let the GEOS team know how their current assets appear on our site and return this back to the GEOS project team to redo the image.

grhhhh committed:
Merge branch 'hackforla:gh-pages' into greenearth-prj-card

aalieu commented:
@grhhhh 

I was assigned to review this PR I think because there was a new commit after the previous review from @blakes24. I looked through the files changed and viewed the feature on my own computer but I didn't see any changes from what was requested in the last review by Blakely so I will hold off on reviewing for now until I hear back from you!

grhhhh committed:
adds readme, prj overview, hero image and prj leaders' gh avatars

grhhhh committed:
Merge branch 'hackforla:gh-pages' into greenearth-prj-card

grhhhh committed:
Merge branch 'greenearth-prj-card' of https://github.com/grhhhh/website into greenearth-prj-card

-m "msg"

grhhhh commented:
@blakes24 @Aveline-art @aalieu Thanks for your feedback. I've made the following changes in my most recent commit:

- added the `green-earth-os-hero.png` to the `green-earth-os.md` file
- added the project leaders' gh avatar images
- added the project overview and readme

I am waiting to receive a new `green-earth-os.png` image with the correct size ratio from the GEOS team. I've also asked for their input regarding the background image of the project page which is currently a blank green background.

grhhhh committed:
updated green-earth-os.png with corrected ratio

blakes24 reviewed (approved):
Nice job with all the changes. The card image looks good and all the links are present. Everything looks great.

macho-catt reviewed (changes requested):
Good job @grhhhh. You're almost there! However, I have a simple change request. Is it possible to put a copy of the project's pdf (GreenEarthOS one sheet.pdf) under assets/pdfs, then update the md file so that both overviews on the project card and project page open a new tab with that pdf (instead of currently opening as a github page with a pdf file)?

We are currently in the process of making new issues to restructure our pdf files. Thanks!

Aveline-art commented:
@macho-catt That's out of the purview of this issue. Let's do the pdf migration as a separate issue, so that @grhhhh would not have to keep working on an issue as requirements change. If everything else is ready, consider merging this.

macho-catt reviewed (approved):
Then this is good to go. GJ!
By the way, there is a separate issue on the projects page that I will create later where url with spaces do not link properly under the a tag. The fix is to enclose the value of the href parameter with double quotes. This seems to be a separate issue from this.
